### ISSUE TITLE
What is the license of Ubuntu 20 04 LTS WSL？

### ISSUE THREAD

**Issue by NewtMa** [microsoft/WSL#7247]
### Discussed in https://github.com/microsoft/WSL/discussions/7184

<div type='discussions-op-text'>

<sup>Originally posted by **NewtMa** July 15, 2021</sup>
I want to use WSL for my daily development and production. I asked my manager and he asked what kind of license did WSL use. And I didn't find a answer on the Internet. I googled and only found out that there was this WSL for Enterprise thing.

Could anyone help me with this? Any help would be appreciated.</div>

**Comment by benhillis** [comment]
WSL itself is part of Windows, so follows the same license as the rest of Windows.  The distributions on the store are owned by their respective companies (Canonical for Ubuntu), so that's up to them.

**Comment by Amondale** [comment]
I found licensing info for the two more popular "flavors" of Linux that you can run with WSL/WSL2/WSLg:

- https://www.kali.org/docs/policy/EULA.txt
- https://ubuntu.com/licensing

Both distros make it pretty clear when you are installing components that aren't considered "Open Source".  https://www.techrepublic.com/blog/linux-and-open-source/solving-the-open-source-vs-proprietary-driver-debate-in-the-linux-kernel/ explains the debate about whether "proprietary drivers" are acceptable in the Linux kernel. Not sure if WSL contains proprietary drivers in its kernel.

**Comment by NewtMa** [comment]
@benhillis @Amondale So sorry for the late reply. Thanks for the great info. I have found a [Ubuntu WSL wiki](https://ubuntu.com/wsl) saying that Canonical offered an enterprise support of Ubuntu on WSL, so I'm guessing there may be some legal problem if you use the free one.

I also happened to find out that there was a new windows terminal on Microsoft Store [source link](https://ubuntu.com/blog/ubuntu-on-wsl-2-is-generally-available), if anyone is interested.
